src/renderer/entities/wallet/lib/account-utils.ts

```diff
@@ -15,33 +14,45 @@ import type {
   WalletConnectAccount,
   ProxiedAccount,
   Wallet,
+  Chain,
+  ChainId,
 } from '@shared/core';
 import { walletUtils } from './wallet-utils';
+import { networkUtils } from '../../network';
 
 export const accountUtils = {
```

Comment:
We clearly should do something with all these functions (obviously not in this PR)
Seems to me if we check so much stuff regarding account, then it's interface is badly designed
Moving accounts into wallet might help